### ISSUE TITLE
What happened to the story of eve

### ISSUE THREAD

**Issue by Kristian8606** [ebaauw/homebridge-hue#730]
After updating the application, the motion sensors look strange.

![4D43E3C1-E98B-42D6-A467-62B7D597092E](https://user-images.githubusercontent.com/34716572/87240458-78a17200-c422-11ea-8864-2127fc6e47a3.png)

**Comment by ebaauw** [comment]
Which application did you update?  Eve?  Are you running iOS 14 beta?  Hoe does it show on other devices?

**Comment by Kristian8606** [comment]
I updated the eve app and since then it looks this way with iOS 13 as well as iOS 14 beta.  this is what only the motion sensors that have a light sensor look like

**Comment by ebaauw** [comment]
What version did you update to?  I haven’t seen a new version released recently.  Did you reboot your iDevice after the update?

**Comment by Kristian8606** [comment]
Version 4.3.1 from 3 weeks ago.  today a new sensor arrived and i added it to deconz i rebooted and it also looks like this

**Comment by Kristian8606** [comment]
Only I have this problem?

**Comment by ebaauw** [comment]
I've been running Eve 4.3.1 (7211) since it came out and have never seen the above.  If it's on more than one iDevice, it's likely not an issue with Eve, but with how deCONZ exposes the sensor.  What type of sensor is this?  Could you please attach the debug dump file?

**Comment by Kristian8606** [comment]
[homebridge.log.gz](https://github.com/ebaauw/homebridge-hue/files/4908373/homebridge.log.gz)
[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/4908374/homebridge-hue.json.gz)

**Comment by ebaauw** [comment]
Which is the sensor you added?

**Comment by Kristian8606** [comment]
Motion sensor test

**Comment by ebaauw** [comment]
And it one that one that looks funny in Eve, or also the other two motion sensors?

I don't see anything odd in the dump file, nor in the log file.  However, you didn't run `homebridge -D` when capturing it, so the debug info is missing.  Could you try again, please.

Also, could you post a screenshot of the _Names_ screen in Eve, under the config settings for the motion sensor?  I should show a motion sensor and a light sensor.

**Comment by Kristian8606** [comment]
[homebridge.log.gz](https://github.com/ebaauw/homebridge-hue/files/4908389/homebridge.log.gz)

sorry this is file

**Comment by Kristian8606** [comment]
![38E03DDB-A615-4F2B-8E7A-E9BDFE497BFF](https://user-images.githubusercontent.com/34716572/87245222-5886a880-c44c-11ea-8788-cb0666c2676d.png)

**Comment by ebaauw** [comment]
This looks OK: one accessory with two services.
```
[7/12/2020, 2:24:41 PM] [Hue] Phoscon-GW: 00158D00047B48D6: LUMI lumi.sensor_motion.aq2 "Motion Sensor test"
[7/12/2020, 2:24:41 PM] [Hue] Phoscon-GW: /sensors/29: ZHAPresence "Motion Sensor test"
[7/12/2020, 2:24:41 PM] [Hue] Phoscon-GW: /sensors/32: ZHALightLevel "Motion Sensor test"
```

I cannot find anything wrong with Homebridge Hue.

That's the _Accessory Details_ screen.  I need the _Names_ screen; second line (under _Room_) in the config screen, with the Siri symbol.

**Comment by Kristian8606** [comment]
![BC95F048-2C65-46EB-8AE0-B063D91E7F38](https://user-images.githubusercontent.com/34716572/87245276-d9de3b00-c44c-11ea-8227-7bd86568bad7.png)

**Comment by ebaauw** [comment]
Hm, looks like it duplicates the light sensor's characteristics as well.  Does the sensor show correctly in Home?  The other motion sensors look OK, don't they?

**Comment by Kristian8606** [comment]
No motion sensor looks good

**Comment by Kristian8606** [comment]
![73E01001-76BA-4178-B729-7707C2496299](https://user-images.githubusercontent.com/34716572/87245392-920be380-c44d-11ea-9a3a-6f23082ad693.jpeg)

**Comment by ebaauw** [comment]
That's looking good as well.

I think it's a bug in Eve, somehow its memory got corrupted.  I don't know what caused it.

I would suggest to uninstall Eve, reboot the iDevice and re-install Eve.  Your HomeKit config should be safe, but the Eve-specific config will be lost.  Not sure about the recent Eve feature to sync the config between iDevices; you might need to do this on all devices simultaneously.

Before trying that, you could try and remove the history for the motion sensors from Eve, and reboot the iDevice.  If that doesn't help, remove the history again (assuming it loaded the history preserved by Homebridge Hue); stop Homebridge, delete the corresponding history files from `accessories`, reboot the iDevice and restart Homebridge.

**Comment by fullphat** [comment]
![image](https://user-images.githubusercontent.com/11242840/87246918-63d7d500-c448-11ea-96a5-adb9d9f2945f.png)

From my Eve app.  Only one actual Eve sensor; the others are Yale alarm sensors.  This is Eve 4.3.1 (7211).

**Comment by Kristian8606** [comment]
I deleted the sensor from deconz I deleted the history from homebridge -> restart I uninstalled eve I added the sensor -> I restarted then I installed eve but it didn't help

**Comment by Kristian8606** [comment]
The problem seems to have been resolved since the last update 4.4 of eve
